### PR TITLE
Adds Match With More Than 2 Minutes...

### PR DESCRIPTION
**Pull request by rafaelsachetto**
Adds Matches With Less Than 2 Minutes
Changes output from MaxKills to Max Kills In A Match
Removes Time Survived In Minutes

### CODE REVIEW

File: pubg.go [hunk 1]
```diff
@@ -183,13 +187,13 @@ func (s Stats) report(totalPlays int) {
 	fmt.Println("Kill Streaks:", s.KillStreaks)
 	fmt.Println("Kills:", s.Kills)
 	fmt.Println("Longest Kill:", s.LongestKill)
-	fmt.Println("MaxKills:", s.MaxKills)
+	fmt.Println("Max Kills In A Match:", s.MaxKills)
+	fmt.Println("Match With More Than 2 Minutes:", s.MatchWithMoreThan2Minutes)
```
Comment (theodorosidmar) [inline pubg.go]: Since we're using plural, how about calling this "Match**es** With More Than 2 Minutes".

File: pubg.go [hunk 2]
```diff
@@ -183,13 +187,13 @@ func (s Stats) report(totalPlays int) {
...
 	fmt.Println("Revives:", s.Revives)
 	fmt.Println("Road Kills:", s.RoadKills)
 	fmt.Println("Suicides:", s.Suicides)
 	fmt.Println("Team Kills:", s.TeamKills)
 	fmt.Println("Time Survived In Hours:", s.TimeSurvived/60/60)
-	fmt.Println("Time Survived In Minutes:", s.TimeSurvived/60)
```
Comment (theodorosidmar) [inline pubg.go]: Why? Don't you think this info more relevant than `Time Survived In Hours`?

File: pubg.go (hunk 1 above)
Comment (theodorosidmar) [inline pubg.go]: What if we also shows `Matches With Less Than 2 Minutes`? That'd just be `Matches` minus `Matches With More Than 2 Minutes` 🤔

File: pubg.go (hunk 2 above)
Comment (rafaelsachetto) [reply pubg.go]: Because we usually plays so much hours I've decided remove this data

File: pubg.go (hunk 1 above)
Comment (rafaelsachetto) [reply pubg.go]: I've noticed that after

**Pull request merged**